nightwatch/features/inoutcommission.feature

```diff
@@ -0,0 +1,95 @@
+Feature: Bed commission state changing events
+
+  Background:
+    Given I am a logged in user
+    And There are no existing centres
+    And The following centres exist:
+      | name | male_capacity | male_cid_name  |
+      | one  | 1000          | oneman,onebman |
+    Given I am on the wallboard
+    And The Centre "one" should show the following under "Male":
+      | Contractual Capacity | 1000 |
+
+  Scenario: Bed going out and back in commission
+    Given I submit the following "out commission" event:
+      | centre    | one   |
+      | timestamp | now   |
+      | bed_ref   | abc6  |
+      | gender    | m     |
+      | reason    | Other |
+    Then The Centre "one" "should" show the following "Reasons" under "Male" "Beds out of commission":
+      | Other | 1 |
+    When I submit the following "in commission" event:
+      | centre    | one  |
+      | timestamp | now  |
+      | bed_ref   | abc6 |
+    Then The Centre "one" "should not" show the following "Reasons and Counts" under "Male" "Beds out of commission":
+      | Other | 0 |
+
+
+  Scenario: Bed going out and back in commission out of order
+    Given I submit the following "in commission" event:
+      | centre    | one  |
+      | timestamp | now  |
+      | bed_ref   | abc6 |
+    And I submit the following "out commission" event:
+      | centre    | one       |
+      | timestamp | 1 day ago |
+      | bed_ref   | abc6      |
+      | gender    | m         |
+      | reason    | Other     |
+    Then The Centre "one" "should not" show the following "Reasons and Counts" under "Male" "Beds out of commission":
+      | Other | 0 |
```

Comment:
you won't actually test anything in this, since it'll execute faster than the event could conceivably reach the front end